src/test/resources/getToken.sh

```
@@ -31,7 +31,7 @@ fi
 getToken() {
 	echo "Get token for user $username on $url"
 	access_token_pattern='"access_token":"([^"]+)"'
-	response=$(curl -v -s -X POST -d "username="$username"&password="$password"&grant_type=password" $url:2002/auth/token)
+	response=$(curl -s -X POST -d "username="$username"&password="$password"&grant_type=password" $url:2002/auth/token)
```

Shellcheck

[warning] The surrounding quotes actually unquote this. Remove or escape them.

(SC2027)